lib/reek/cli/options.rb

```diff
@@ -57,8 +57,8 @@ def parse
       private
 
       # :reek:UtilityFunction
-      def color_support?
-        $stdout.tty?
+      def piped_output?
```

Comment:
I think the name is confusing. In the cucumber features above nothing gets piped, at least not in the way I - and probably others - understand pipes. Basically the way we run Reek in our features is the default way that involves no pipe. Maybe rename this to "non_tty_output" and put an explanatory comment above?

I really have a shitty understanding of the tty concept which comes from medieval times and I really do not want to read [tty demystified](http://www.linusakesson.net/programming/tty/) again in my life. Like, ever :-/